### ISSUE TITLE
Site support request: memritv.org

### ISSUE THREAD

**Issue by arisboch** [rg3/youtube-dl#7403]
Could you add support for downloading videos from the video site of MEMRI?
Here's a few links to random videos from the site:

http://www.memritv.org/clip/en/5148.htm
http://www.memritv.org/clip/en/5142.htm
http://www.memritv.org/clip/en/5141.htm

Note that every or almost every video has also a transcript of the speech linked to it, it'd be nice, if you could just integrate it into the .description file or something like that.

**Comment by jaimeMF** [comment]
See https://github.com/rg3/youtube-dl/pull/6210#issuecomment-137930881.

**Comment by arisboch** [comment]
Apparently this problem was resolved, since the Memri TV YouTube channels are up and running again.

**Comment by arisboch** [comment]
Now it's been more than a year and nothing happened. Why not? Their YouTube channel was unblocked again.

**Comment by arisboch** [comment]
Why was this not resolved after almost five years?

**Comment by arisboch** [comment]
It's already more than nine years?!

**Comment by dirkf** [comment]
Here's a current clip: https://www.memri.org/tv/chinese-professor-shen-yi-deepseek-digital-silk-road.

The page is not handled by the Generic extractor but it looks like a simple job to create a site-specific extractor. The [embed page](https://www.memri.org/player/clip/69266/1/1/card) linked in `twitter:player` and elsewhere contains media URLs (RTMP and HLS) in a JS object. Metadata is available in the main page.

**Comment by paypaxx** [comment]
> Apparently this problem was resolved, since the Memri TV YouTube channels are up and running again.

No, it's actually not resolved because the YouTube channel contains only 25% of their actual content. It was blocked and banned in the past and they had to start a new channel.

**Comment by arisboch** [comment]
@paypaxx Oh, c'mon, this is a lame excuse and you know it.
